WooCommerce/Classes/ViewRelated/Products/Cells/ProductsTabProductTableViewCell.swift

```diff
@@ -97,8 +97,6 @@ extension ProductsTabProductTableViewCell {
             selectedProductImageOverlayView?.removeFromSuperview()
             selectedProductImageOverlayView = nil
         }
-        let selectedBackgroundColor = isSelected ? UIColor.primary.withAlphaComponent(0.2): .listForeground(modal: false)
-        backgroundColor = selectedBackgroundColor
```

Comment:
I removed this since we are using `selectedBackgroundView` and `selectedBackgroundView?.backgroundColor`, is that ok?

Reply:
Makes sense!